### DISCUSSION TITLE
Module manager help?

### DISCUSSION THREAD

**Discussion by Y-Perry** [AlexandreRouma/SDRPlusPlus#315] [Q&A]
I tried to use the module manager, but accomplished nothing. Where can I find a help page for the Module manager?

Thanks,

Yonadav

**Comment by AlexandreRouma** [comment]
there is no help page but it's pretty simple:

- Click the - button on a module instance to remove it
- Select a module type in the drop down at the bottom and give the instance a name using the text field next to it, then create that instance by pressing the + button
